### PR TITLE
adding bms paper pub + fix name on PoM pub

### PR DESCRIPTION
**Pull request by vivek-bala**

### CODE REVIEW

**Comment by vivek-bala** [comment]
Hi, I just added the entry for the bms paper with the arxiv link. The paper is submitted and pending acceptance decision. I am not sure what the agreed method was, so for now I have added a 'pending decision' remark on the venue tag. 

Let me know if we have decided on a method to mark that a paper is under review at conference X or if it is not required.

My suggestion would be to separate submitted, accepted, and published papers with a 'status' tag.

File: _publications/balasubramanian2018adaptive.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+title: "Adaptive Ensemble Biomolecular Simulations at Scale"
+collection: publications
+permalink: /publications/balasubramanian2018adaptive
+type: pub
+date: 2018-10-27
+author: "Vivek Balasubramanian, Travis Jensen, Matteo Turilli, Peter Kasson, Michael Shirts and Shantenu Jha"
+venue: "33rd IEEE International Parallel and Distributed Processing Symposium (pending decision)"
```
Comment (iparask) [inline _publications/balasubramanian2018adaptive.md]: Can you please change the venue here from the conference to Arxiv?

**Review by iparask — CHANGES_REQUESTED** [review]
Apart from the requested change everything else seems okay

**Comment by iparask** [comment]
After discussing with @mturilli, we said that as `venue` we will use where the paper is published. If for example that is Arxiv the entry should say Arxiv. The venue can change to a conference/journal only when a paper is been accepted. In the case of a conference, the venue will actually change when attendance is verified by at least one author, because in most cases attendance of at least an author is required for conferences.

Does it help?

**Review by iparask — APPROVED** [review]

**Comment by vivek-bala** [comment]
Yes that makes sense. I guess part of the reason would have been that the venue needs to be updated if the paper isn't accepted and we submit to another venue (and so on till it gets accepted). Changed the venue now. Please review

**Comment by mturilli** [comment]
The notions of submitted, under review, accepted, rejected are not used in the website. A paper has only one state---published---and the venue is proceedings of a conference, journal, arxiv or the lab (in case of a technical paper).

**Pull request merged**